_includes/page_header.html

```diff
@@ -20,7 +20,7 @@
             <a href="https://www.apache.org/events/current-event" target="_blank">Events</a>
             <a href="https://www.apache.org/licenses/" target="_blank">License</a>
             <a href="https://www.apache.org/foundation/thanks.html" target="_blank">Thanks</a>
-            <a href="https://www.apache.org/security/" target="_blank">Security</a>
+            <a href="https://druid.apache.org/docs/latest/operations/security-overview.html" target="_blank">Security</a>
```

Comment:
this is under Apache section. So it makes sense to link to Apache Security page.

Reply:
Do you have an alternative suggestion?

While it's not entirely uncommon for Apache pages to link to project-specific security pages from the 'Security' link in the 'Apache' section (e.g. https://ofbiz.apache.org/, https://pdfbox.apache.org/), I agree it's a bit odd. I'd like to make the project-specific page more prominent, though. Do you think it could be a top-level item?

Reply:
We can put the link in "Learn more" section on home page. @vogievetsky sounds good to you?